File: C/Chapter-07/Question/Question6.c
#include <stdio.h>

int main(){
  int num= 0;
  int sum = 0;
  int total = 0;

  while(total<5){
    num = 0;
    while(num<=0){
      printf("1 이상의 정수 입력: ");
      scanf("%d", &num);
    }
    sum += num;
    total++;
  }

  printf("총합: %d",sum);
  
  return 0;
}

File: Java/Chapter-05/Question/Question6.java
public class Question6 {
  public static void main(String[] args) {
    int sum = 0;
    for(int i = 1; ; i++){
      if(i % 2 != 0){
        sum += i;
      }
      if(sum > 1000){
        System.out.println("sum = " + sum);
        System.out.println("i = " + i);
        break;
      }
    }
  }
}
